### PR TITLE
Homepage component

### PR DESCRIPTION
**Pull request by drhanciles**
#### Description
Implemented `HomePage` component. Also implemented base styling for the component. Initiated test suite for the component and it is passing snapshot test. Corrected issue that was preventing background color throughout the app on this branch as well. 

----
#### Issues # and Action
closes #39 

----
#### TO DO / Noteworthy

----
#### Features
- [ ] New Feature
- [ ] Expand Feature
- [ ] Optimize/Improve Feature

----
#### Testing
- [ ] No Testing
- [ ] Some Testing
- [x] Sufficient Testing

- [ ] Left notes near functions that have not been tested
- [ ] Intentionally skipped / failing / empty tests as a reminder to add/improve these later

- [ ] Tests have been changed
- [ ] Existing tests are now failing / skipped

----
#### Deployment
- [x] This is a complete feature, ready for deployment
- [ ] This is a partial feature, NOT ready for deployment

----
#### API Changes
(please detail in description)
- [ ] New API
- [ ] Existing API, New Endpoint
- [ ] README.md is updated

----
#### Tech Changes
(please describe)
- [ ] New Functional Library
- [ ] New Testing Library
- [ ] New Debugging Tool
- [ ] README.md is updated

### CODE REVIEW

File: src/HomePage/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        <article className="message-buttons-container">
+          <button className="done-button">Done</button>
+          <button className="reminder-button">Remind Me Tomorrow</button>
+        </article>
+        <section className="occasions-container"> 
+        <h3>Upcoming Occasions:</h3>
+          <section className="occasions-week">
+            <article className="occasion">
+              <p className="date-container">02/16/19</p>
+              <p className="occasaion-reminder-content">Celebrate Gina's Birthday</p>
```
Comment (RajaaBoulassouak) [inline src/HomePage/HomePage.js:68]: Is there a reason why some content is having a red background?

**Review by RajaaBoulassouak — COMMENTED** [review]
Great work on the homepage component, Derek! Do you want to update the Issue number in the PR template before I merge it? ( closes #39)?

File: src/HomePage/HomePage.js (hunk 1 above)
Comment (drhanciles) [reply src/HomePage/HomePage.js:68]: i'll get it taken care of now!

Comment (drhanciles) [reply src/HomePage/HomePage.js:68]: issue has been updated.

Comment (RajaaBoulassouak) [reply src/HomePage/HomePage.js:68]: Great! I'll merge everything now.

**Pull request merged**